AstSemantics.md

```diff
@@ -286,10 +236,7 @@ Each function has a *signature*, which consists of:
   * Return types, which are a sequence of local types
   * Argument types, which are a sequence of local types
 
-Note that WebAssembly itself does not support variable-length argument lists
-(aka varargs). C and C++ compilers are expected to implement this functionality
-by storing arguments in a buffer in linear memory and passing a pointer to the
-buffer.
```

Comment:
This is also footnote-like material which feels like it belongs here, though perhaps it could be fully parenthesized to indicate the informative nature.